odoo/addons/base/wizard/base_language_install_views.xml

```diff
@@ -2,34 +2,43 @@
 <odoo>
     <data>
 
+        <record id="view_base_language_switch" model="ir.ui.view">
```

Comment:
Let us stick on guidelines here, like ``base.language.install.view.form.lang.switch`` or something like that ?